tests/common/point_test.cpp

```diff
@@ -1,10 +1,10 @@
-#include "gtest/gtest.h"
+#include <gtest/gtest.h>
 #include "src/core/common/bounding_box.h"
 
 TEST(Point_test, shifting_test) {
-    Point point(21, 37);
+    Point2i point(21, 37);
```

Comment:
You should also test Point2f

Reply:
Lol, like literally no. It's a simple method that does not use mathematical operators that could lead us to unpredictable overflow/rounding turbulences nor it is specialized template.

Reply:
So can I assume that comarch developers don't test their code properly?